### PR TITLE
Adding netcdf_to_tplot functionality, along with merge variable request

### PR DESCRIPTION
**Pull request by jibarnum**
A request was made to make a a netcdf_to_tplot function, like the cdf_to_tplot function. I did that, and while I was at it I also added in a functionality to merge tplot variables together, or read in multiple files into one tplot variable. Let me know if you see anything I should address, or add, or change!

### CODE REVIEW

File: netcdf_to_tplot.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        >>> import pytplot
+        >>> file = "/Users/user_name/goes_files/g15_epead_a16ew_1m_20171201_20171231.nc"
+        >>> pytplot.netcdf_to_tplot(file, prefix='goes_prefix_', plot=True)
+
+    '''
+
+    stored_variables = []
+    global data_quants
+
+    print(data_quants)
```
Comment (bryan-harter) [inline netcdf_to_tplot.py]: Remove print statement

File: netcdf_to_tplot.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            masked_vars[var] = var_filled
+
+        # Most files are from GOES data, which seems to usually have 'time_tag' in them that contain time information.
+        # There is an exception filter below that will allow a user to pick a different time variable if time_tag doesn't exist.
+        if 'time_tag' in vars_and_atts.keys():
+            time_var = file['time_tag']
+            unix_times = change_time_to_unix_time(time_var)
+
+        if 'time_tag' not in vars_and_atts.keys():
+            time_var_exists = input('Is there a differently-named time variable in the variables (y/n)? \nVariable list: {l}'.format(l=vars_and_atts.keys()))
```
Comment (bryan-harter) [inline netcdf_to_tplot.py]: This is great, but can you also add a variable in the function call where someone can specify the name of the time variable there?  

My thinking is if someone comes along with a bunch of netCDF files where the time variables are all named the something weird, they probably don't want to have to go through this process each time.  So they have the option of saying something like " time_var_name='whatever' " when they call the function.

File: netcdf_to_tplot.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                elif time_var_exists != 'y' and time_var_exists != 'n':
+                    print('You did not enter a valid response, please try again.')
+                    # We grabbed whatever the time value is, yay! Time to grab the other variable to save/plot
+                    continue
+                elif time_var_exists == 'n':
+                    print('Without a time variable, this file is not a valid file type for these kinds of plots.')
+                    exit()
+
+        while True:
+            var_of_interest = input('What variable do you want to plot? Please pick from {l}.'.format(l=vars_and_atts.keys()))
```
Comment (bryan-harter) [inline netcdf_to_tplot.py]: I think by default, every variable should be read in

File: pytplot/netcdf_to_tplot.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+        for i,var in enumerate(file.variables):
+            # Here, we are making sure that the variables are time-based, otherwise we don't want to store them as tplot variables!
+            if 'record' in file[var].dimensions[0] or 'time' in file[var].dimensions[0]:
+                # Store the data now, as well as merge variables if that's desired
+                var_name = prefix + var + suffix
+                to_merge = False
+                if (var_name in data_quants.keys() and (merge == True)):
+                    prev_data_quant = data_quants[var_name].data
+                    print(prev_data_quant)
```
[the remark this answers is not in the document]
Comment (jibarnum) [reply pytplot/netcdf_to_tplot.py]: yep, I'll take it out!

File: pytplot/netcdf_to_tplot.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+                masked_vars[var] = var_filled
+
+        # Most files are from GOES data, which seems to usually have 'time_tag' in them that contain time information.
+        # There is an exception filter below that will allow a user to pick a different time variable if time_tag doesn't exist.
+        if 'time_tag' in vars_and_atts.keys():
+            time_var = file['time_tag']
+            unix_times = change_time_to_unix_time(time_var)
+
+        if 'time_tag' not in vars_and_atts.keys():
+            time = input('Please enter time variable name. \nVariable list: {l}'.format(l=vars_and_atts.keys()))
```
[the remark this answers is not in the document]
Comment (jibarnum) [reply pytplot/netcdf_to_tplot.py]: yes, can do. I think I had read what you originally said about this and misunderstood.

[the remark this answers is not in the document]
Comment (jibarnum) [reply pytplot/netcdf_to_tplot.py]: Okay, what I push up will reflect our offline conversation; There will be an optional argument in the function call to entire the time variable name, however, if the user does not do that, then they will be asked to input the time variable.

**Comment by jibarnum** [comment]
I believe I addressed all the comments made on this ticket. Should we begin the merging process?

**Comment by bryan-harter** [comment]
Looks good!  Feel free to merge away

**Pull request merged**